packages/core/src/api/blockManipulation/commands/mergeBlocks/mergeBlocks.test.ts

```diff
@@ -59,7 +59,7 @@ describe("Test mergeBlocks", () => {
     expect(getEditor().document).toMatchSnapshot();
   });
 
-  it.skip("Inline content & table content", () => {
+  it("Inline content & table content", () => {
```

Comment:
the snapshots are difficult to review; what happens in these 2 actual cases / what has changed?